cpufreq/src/cpufreq-selector/cpufreq-selector-sysfs.c

```diff
@@ -27,18 +27,22 @@
 #include "cpufreq-selector-sysfs.h"
 
 struct _CPUFreqSelectorSysfsPrivate {
-	GList *available_freqs;
-	GList *available_govs;
+    GList *available_freqs;
+    GList *available_govs;
 };
 
-static void     cpufreq_selector_sysfs_finalize      (GObject                   *object);
+static void
+cpufreq_selector_sysfs_finalize (GObject                   *object);
```

Comment:
cpufreq_selector_sysfs_finalize columns are not aligned with cpufreq_selector_sysfs_set_frequency, cpufreq_selector_sysfs_set_governor